spec/aho_corasick_matcher_spec.rb

```diff
@@ -78,4 +78,12 @@
       ).to eq(%w(Test TestString String Test))
     end
   end
+
+  describe '#inspect' do
+    let(:dict) { ('aaa'...'bbb').to_a }
+
+    it 'does not include instance variables to prevent stack explosions' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.